src/globus_cli/commands/flows/run/resume.py

```diff
@@ -68,6 +62,41 @@ def resume_command(
     display(res, fields=fields, text_mode=TextMode.text_record)
 
 
+def check_inactive_reason(
+    login_manager: LoginManager,
+    gare: GlobusAuthRequirementsError | None,
+) -> None:
+    if gare is None:
+        return
+    if gare.authorization_parameters.required_scopes:
+        consent_required = not _has_required_consent(
+            login_manager, gare.authorization_parameters.required_scopes
+        )
+        if consent_required:
+            raise CLIAuthRequirementsError(
+                "This timer is missing a necessary consent in order to resume.",
+                gare=gare,
+            )
+
+    # at this point, the required_scopes may have been checked and satisfied
+    # therefore, we should check if there are additional requirements other than
+    # the scopes/consents
+    unhandled_requirements = set(gare.authorization_parameters.to_dict().keys()) - {
+        "required_scopes",
+        # also remove 'message' -- not a 'requirement'
+        "session_message",
+    }
+    # reraise if anything remains after consent checking
+    # this ensures that we will reraise if we get an error which contains
+    # both required_scopes and additional requirements
+    # (consents may be present without session requirements met)
+    if unhandled_requirements:
+        raise CLIAuthRequirementsError(
+            "This timer needs strong authentication in order to resume.",
```

Comment:
~timer~ run

Separately, regarding phrasing -- I don't think the line "strong authentication" communicates what the run needs to resume.

Reply:
Good catches on timer/run confusion by me!

What should we call it? It's part of a larger error message which instructs the user that they need to run `globus session update FOO BAR BAZ`.
In that context, it renders like so:
```
The resource you are trying to access requires you to re-authenticate.
This timer needs strong authentication in order to resume.

Please run

    globus session update globus.org

to re-authenticate with the required identities

* * *

After updating your session, resume the timer with

    globus timer resume --skip-inactive-reason-check 35575759-e72a-4be5-be9f-7df73015382b

```

We can adjust any part of the rendering here as seems suitable, within some limits.
But this string is just parametrizing that second line to be specific to the context in which the exception was raised.

Reply:
Oh! In that broader context it's much clearer what's happening. Nevertheless, "strong authentication" seems non-ideal. Perhaps:

```
This timer has additional authentication requirements that must be met.
```

?

Reply:
Yeah, let's go with that and I'll plumb it so that it replaces the first line of the messaging rather than supplementing it.

Reply:
I'm going to mark this resolved by the recent changes, which should align well with this discussion.